api_server/src/lib.rs

```diff
@@ -39,6 +40,11 @@ use vmm::default_syscalls;
 use vmm::vmm_config::instance_info::InstanceInfo;
 use vmm::VmmAction;
 
+/// One shot channel used to send a request.
+pub type OutcomeSender = oneshot::Sender<vmm::VmmRequestOutcome>;
+/// One shot channel used to receive a response.
+pub type OutcomeReceiver = oneshot::Receiver<vmm::VmmRequestOutcome>;
```

Comment:
Why not RequestSender and ResponseReceiver ?

Reply:
They could be `ResponseSender` and `ResponseReceiver`, but no request, they're the two ends of a channel that delivers `Response`s or `VmmRequestOutcome`s.

Reply:
It's a little bit confusing since the comments actually tell a story with no outcomes.

Reply:
Sure. Can change `OutcomeSender` and `OutcomeReceiver` to `ResponseSender` respectively `ResponseReceiver` since these are now ApiServer owned.

**But FYI** `VmmRequestOutcome` was changed to this some time ago from `VmmResponse` so that it doesn't have `HTTP` naming since it should be able to be driven from the outside in many ways not necessarily through a HTTP ApiServer.
Now I don't personally agree, nor disagree with that decision. Just bringing you up to speed with a decision taken a while back in the team.